### PR TITLE
Feature/remote atf

### PR DESCRIPTION
**Pull request by ghost**
ApplinkConnection module with Send methon and message type enum, and changes connecttest

### CODE REVIEW

File: modules/applink_connection.lua [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+local remote_mq_utils = require("remote/remote_mq_utils")
+
+local ApplinkConnection = {
+  mt = { __index = {} }
+}
+
+--- Appink messege type enumeration
+ApplinkConnection.SDLMsgType ={
+	SDL_MSG_SDL_START = 0x10,
```
Comment (aderiabin) [inline modules/applink_connection.lua]: @AStarodub Please fix code indentation.

File: modules/connecttest.lua [hunk 2]
```diff
@@ -45,11 +45,11 @@ local FAILED = expectations.FAILED
 if config.remoteConnection.enabled then
   Test.remoteConnection = remote_connection.RemoteConnection(config.remoteConnection.url, config.remoteConnection.port)
   Test.remoteConnection:Connect()
-
   Test.remoteUtils = {}
   Test.remoteUtils.file = remote_file_utils.RemoteFileUtils(Test.remoteConnection)
   Test.remoteUtils.app = remote_application_utils.RemoteAppUtils(Test.remoteConnection)
-  Test.remoteUtils.mq = remote_mq_utils.RemoteMqUtils(Test.remoteConnection)
+  Test.applinkConnection = applink_connection.ApplinkConnection(Test.remoteConnection)
+--  Test.remoteUtils.mq = remote_mq_utils.RemoteMqUtils(Test.remoteConnection) todo AStarodub recheck!
```
Comment (aderiabin) [inline modules/connecttest.lua]: @AStarodub Please remove commented code.

**Review by aderiabin — CHANGES_REQUESTED** [review]

File: modules/applink_connection.lua (hunk 1 above)
Comment (ghost) [reply modules/applink_connection.lua]: Done

File: modules/connecttest.lua (hunk 2 above)
Comment (ghost) [reply modules/connecttest.lua]: Done

**Review by aderiabin — APPROVED** [review]

**Pull request merged**